packages/ember-runtime/lib/mixins/controller.js

```diff
@@ -44,6 +44,5 @@ export default Mixin.create(ActionHandler, ControllerContentModelAliasDeprecatio
   /**
     @private
   */
-  content: alias('model')
-
+  content: deprecatingAlias('model', { id: 'ember-runtime.controller-content', until: '2.16.0' })
```

Comment:
We also need to add a deprecation guide entry to the website and add a `url` property with the link.